rtsp.go

```diff
@@ -58,7 +59,8 @@ func init() {
 
 // Config are the config attributes for an RTSP camera model.
 type Config struct {
-	Address          string                             `json:"rtsp_address"`
+	RTSPAddress      string                             `json:"rtsp_address"`
+	MulticastAddress string                             `json:"multicast_address"`
```

Comment:
Config attribute additions to something like RTSP cam will need to go through a scope before being merged. Unfortunately.

Reply:
Ah all good. I soft reverted to before that commit so we can have the diff if we need it.